share/completions/climate.fish

```diff
@@ -0,0 +1,81 @@
+complete -f -n '__fish_climate_has_no_argument' -c climate -a countdown -d 'A countdown timer'
+complete -f -n '__fish_climate_has_no_argument' -c climate -a stopwatch -d 'A stopwatch'
+complete -f -n '__fish_climate_has_no_argument' -c climate -a ix -d 'Pipe output to ix.io'
+
+# Files
+complete -n '__fish_climate_has_no_argument' -c climate -a biggest-files -d 'Find the biggest files recursively'
+complete -n '__fish_climate_has_no_argument' -c climate -a biggest-dirs -d 'Find the biggest directories'
+complete -n '__fish_climate_has_no_argument' -c climate -a dir-size -d 'Find directory size'
+complete -n '__fish_climate_has_no_argument' -c climate -a remove-empty-dirs -d 'Remove empty directories'
+complete -n '__fish_climate_has_no_argument' -c climate -a extract -d 'Extract any given archive'
+complete -n '__fish_climate_has_no_argument' -c climate -a find-duplicates -d 'Report duplicate files in a directory'
+complete -n '__fish_climate_has_no_argument' -c climate -a count -d 'Count the number of occurences'
+complete -n '__fish_climate_has_no_argument' -c climate -a monitor -d 'Monitor file for changes'
+complete -f -n '__fish_climate_has_no_argument' -c climate -a grep -d 'Search for the given pattern recursively'
+complete -f -n '__fish_climate_has_no_argument' -c climate -a replace -d 'Replace all occurences'
+complete -f -n '__fish_climate_has_no_argument' -c climate -a ramfs -d 'Create a ramfs of size (in MB) at path'
+
+# Network
+complete -f -n '__fish_climate_has_no_argument' -c climate -a speedtest -d 'Test your network speed'
+complete -f -n '__fish_climate_has_no_argument' -c climate -a local-ip -d 'Retrieve your local ip address'
+complete -f -n '__fish_climate_has_no_argument' -c climate -a is-online -d 'Verify if you\'re online'
+complete -f -n '__fish_climate_has_no_argument' -c climate -a public-ip -d 'Retrieve your public ip address'
+complete -f -n '__fish_climate_has_no_argument' -c climate -a ports -d 'List open ports'
+complete -f -n '__fish_climate_has_no_argument' -c climate -a hosts -d 'Edit the hosts file'
+complete -f -n '__fish_climate_has_no_argument' -c climate -a http-server -d 'http-server serving the current directory'
+complete -f -n '__fish_climate_has_no_argument' -c climate -a is-up -d 'Determine if server is up'
+
+# SSH
+complete -f -n '__fish_climate_has_no_argument' -c climate -a download-file -d 'Download file from server'
+complete -f -n '__fish_climate_has_no_argument' -c climate -a download-dir -d 'Download dir from server'
+complete -n '__fish_climate_has_no_argument' -c climate -a upload -d 'Upload to server'
+complete -f -n '__fish_climate_has_no_argument' -c climate -a ssh-mount -d 'Mount a remote path'
+complete -n '__fish_climate_has_no_argument' -c climate -a ssh-unmount -d 'Unmount a ssh mount'
+
+# git
+complete -f -c climate -n '__fish_is_git_repository and __fish_climate_has_no_argument' -a undo-commit -d 'Undo the latest commit'
```

Comment:
This doesn't work because of a missing ";" - `__fish_is_git_repository; and`. The way it is now, `__fish_is_git_repository` gets the rest as argument, and `__fish_climate_has_no_argument` is never executed.